src/main/java/seedu/address/logic/commands/AddCommand.java

```diff
@@ -69,4 +102,133 @@ public String toString() {
                 .add("toAdd", toAdd)
                 .toString();
     }
+
+    /**
+     * Stores the details to edit the task with. Each non-empty field value will replace the
+     * corresponding field value of the task.
+     */
+    public static class AddTaskDescriptor {
+        private Description description;
+        private Deadline deadline;
+        private Priority priority;
+        private Set<Member> members;
+        private Status status;
+        private Note note;
```

Comment:
Do we need these 2 under AddTaskDescriptor since status and note of AddTaskDescriptor is not used in creating a new task?